pkg/handlers/generic/mutation/encryption/encryptionprovider_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 Nutanix. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package encryption
```

Comment:
nit: please rename the package and the file names
```suggestion
package encryptionatrest
```

Reply:
fixed https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/commit/89d1408e8533448aa0168d904a7add4896d6150b